components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -289,6 +289,9 @@ private String getActionTypeFromPath(String actionType) throws ActionMgtClientEx
      */
     private void validateMaxActionsPerType(String actionType, String tenantDomain) throws ActionMgtException {
 
+        if (Action.ActionTypes.Category.IN_FLOW.equals(Action.ActionTypes.valueOf(actionType).getCategory())) {
+            return;
+        }
```

Comment:
Let's add a comment mentioning for IN_FLOW action types multiple actions are allowed as each action will have an associated executor in the flow (e.g, an authenticator)

Reply:
Addressed